internal/featuresext/cpp_features.protoset

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Is it just the size of this file that makes git not classify it as binary? The other java_features.protoset is binary.

Reply:
I suspect maybe it look to be legal UTF8? It looks like there may be some un-printable characters in here, but maybe it's weighted on a ratio of printable to un-printable/control that this file doesn't quite exceed?

Reply:
FWIW, I've seen `git` classify proto source files (which are just text) as binary. I am not certain what its heuristic is.